### PR TITLE
added fourthwall

### PR DESCRIPTION
**Pull request by AbhinavS96**
#5439

### CODE REVIEW

File: src/technologies/f.json [hunk 1]
```diff
@@ -1068,6 +1068,15 @@
     "scriptSrc": "foursixty\\.com",
     "website": "https://foursixty.com/"
   },
+  "Fourthwall": {
+    "cats": [
+      6
+    ],
+    "description": "Fourthwall helps to create and launch a branded website,",
+    "icon": "fourthwall.png",
+    "scriptSrc": "cdn\\.fourthwall\\.com/shop-event-listener/shop-events-listener\\.js",
```
Comment (nurbek91) [inline src/technologies/f.json:1077]: ` "scriptSrc": "\\.fourthwall\\.com/",`

File: src/technologies/f.json [hunk 2]
```diff
@@ -1068,6 +1068,15 @@
     "scriptSrc": "foursixty\\.com",
     "website": "https://foursixty.com/"
   },
+  "Fourthwall": {
+    "cats": [
+      6
+    ],
+    "description": "Fourthwall helps to create and launch a branded website,",
```
Comment (nurbek91) [inline src/technologies/f.json:1075]: . at the end

File: src/technologies/f.json (hunk 1 above)
Comment (nurbek91) [reply src/technologies/f.json:1077]:
```
"meta": {
      "version": "^(.+)$\\;version:\\1\\;confidence:0"
    },
"js": {
      "fourthwallTheme": "",
     "FourthwallAnalytics": ""
    },
 "pricing": [
      "payg"
    ],
  "saas": true,
```

File: src/technologies/f.json [hunk 3]
```diff
@@ -1068,6 +1068,15 @@
...
+    "icon": "fourthwall.png",
```
Comment (nurbek91) [inline src/technologies/f.json:1076]: [Fourthwall.svg.zip](https://github.com/AliasIO/wappalyzer/files/7770530/Fourthwall.svg.zip)
svg instead png

**Review by nurbek91 — COMMENTED** [review]
Hey there, thanks for PR! Little changes required.
examples:
https://imbrandonfarris.com/
https://shop.harrymackofficial.com/
https://ellenandbrian.com/

File: src/technologies/f.json (hunk 3 above)
Comment (AbhinavS96) [reply src/technologies/f.json]: updated!

**Comment by baseplate-admin** [comment]
Hi, @AliasIO, 

It seems you Didn't merge this repo. Any reasons?

**Comment by AliasIO** [comment]
Not all of @nurbek91's suggestions had been implemented yet. I've done this just now.

**Comment by baseplate-admin** [comment]
Thanks 🍡

**Pull request merged**